packages/blocks/src/store/selectors.js

```diff
@@ -95,6 +95,44 @@ export function getBlockVariations( state, blockName, scope ) {
 	} );
 }
 
+/**
+ * Returns the active block variation for a given block based on its attributes.
```

Comment:
I think we can expand a bit on the description.

Reply:
Addressed in the latest commit.